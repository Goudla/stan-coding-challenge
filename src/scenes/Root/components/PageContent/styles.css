.page-content {
  padding-top: 60px;
  padding-bottom: 215px;
}
