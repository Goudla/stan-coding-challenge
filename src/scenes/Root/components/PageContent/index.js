import * as React from 'react';

// Styles
import './styles.css';

type Props = {
  children: React.Node
};

class PageContent extends React.PureComponent<Props> {
  render() {
    const { children } = this.props;

    return (
      <div className="page-content">
        {children}
      </div>
    );
  }
}

export default PageContent;
