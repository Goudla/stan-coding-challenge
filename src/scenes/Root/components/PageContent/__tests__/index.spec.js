import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';

// Scenes
import PageContent from '../';

describe('<PageContent />', () => {
  it('Should render', () => {
    const renderer = new ShallowRenderer();
    renderer.render(<PageContent>Test</PageContent>);
    const result = renderer.getRenderOutput();
    expect(result).toMatchSnapshot();
  });
});
