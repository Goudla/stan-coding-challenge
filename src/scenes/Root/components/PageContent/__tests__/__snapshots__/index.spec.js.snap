// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PageContent /> Should render 1`] = `
<div
  className="page-content"
>
  Test
</div>
`;
